eks_fargate/README.md

```diff
@@ -416,7 +416,7 @@ Need help? Contact [Datadog support][22].
 [19]: http://docs.datadoghq.com/agent/cluster_agent/event_collection
 [20]: https://docs.datadoghq.com/help
 [21]: https://app.datadoghq.com/containers
-[22]: https://app.datadoghq.com/process
+[22]: https://docs.datadoghq.com/integrations/ecs_fargate/#?tab=fluentbitandfirelens
```

Comment:
There are 3 different links pointing to `[22]`. We should update all accordingly.